react/features/base/app/components/BaseApp.js

```diff
@@ -129,6 +131,7 @@ export default class BaseApp extends Component<*, State> {
                 <I18nextProvider i18n = { i18next }>
                     <Provider store = { store }>
                         <Fragment>
+                            { interfaceConfig.SHOW_CHROME_EXTENSION_BANNER && <ChromeExtensionBanner /> }
```

Comment:
Have you tested your changes on mobile? I think this will break mobile. You need to handle interface config as APP.
```
typeof interfaceConfig !== 'undefined' && interfaceConfig.SHOW_CHROME_EXTENSION_BANNER && <ChromeExtensionBanner />
```

Reply:
done, added the check for undefined.